src/content/1.7/development/compile-assets.md

```diff
@@ -85,4 +85,11 @@ The "watch" build will optimize your assets for development. Please remember to
 
 If `npm install` fails with error: `Failed at the ... postinstall script.`:
 
-You may be using node version 9 or 10, you should use node version 8.
+You may be using an old version or one we don't support yet. Also, while some folders (like new-theme, default, classic) support node 12.x, some may not because they are not updated yet and are waiting for a pull request to be merged.
```

Comment:
We shouldn't talk about the waiting pull request.
We should split the node versions for the classic and the admin themes.

Reply:
Then, I'll just put this in WIP and wait for the PR to be merged (this should be fast..)